src/ceph-volume/ceph_volume/devices/raw/list.py

```diff
@@ -69,7 +69,7 @@ def __init__(self, argv):
     def generate(self, devs=None):
         logger.debug('Listing block devices via lsblk...')
         info_devices = disk.lsblk_all(abspath=True)
-        if devs is None or devs == []:
+        if not devs or not any(devs):
```

Comment:
This has the same problem as the previous version: it doesn't handle the case where `devs` contains a non-None device name which does not exist in the system.